shared-helpers/src/locales/general.json

```diff
@@ -608,6 +617,259 @@
+  "help.housingHelp.counseling.santaClaraPS": "Project Sentinel",
+  "help.housingHelp.counseling.santaClaraPSinfo": " - Project Sentinel provides counseling and services for tenants experiencing housing discrimination.",
+  "help.housingHelp.vouchers.title": "Section 8 Housing Vouchers and Units",
+  "help.housingHelp.vouchers.header": "The Section 8 program includes both Housing Choice Vouchers, which are subsidies that can be used for any housing unit, and Project Based Vouchers, which are tied to particular units.  Both are issued by local public housing agencies known as housing authorities. Some housing authorities also own and operate affordable housing units.",
+  "help.housingHelp.vouchers.alamedaHAcounty": "Housing Authority County of Alameda",
+  "help.housingHelp.vouchers.alamedaHAcity": "Housing Authority of the City of Alameda",
+  "help.housingHelp.vouchers.berkeleyHA": "Berkeley Housing Authority",
+  "help.housingHelp.vouchers.livermoreHA": "Housing Authority of the City of Livermore",
+  "help.housingHelp.vouchers.oaklandHA": "Oakland Housing Authority",  
+  "help.housingHelp.vouchers.pleasantonHA": "Housing Authority of the City of Pleasanton",
+  "help.housingHelp.vouchers.richmondHA": "Housing Authority of the City of Richmond",
+  "help.housingHelp.vouchers.pittsburghHA": "City of Pittsburg Housing Authority",
+  "help.housingHelp.vouchers.marinHA": "Housing Authority of the County of Marin",
+  "help.housingHelp.vouchers.napaHA": "Housing Authority of the City of Napa",
+  "help.housingHelp.vouchers.sfHA": "Housing Authority of the City and County of San Francisco",
+  "help.housingHelp.vouchers.sanMateoHA": "Housing Authority of the County of San Mateo",
+  "help.housingHelp.vouchers.southSFHA": "City of South San Francisco Housing Authority",
+  "help.housingHelp.vouchers.santaClara": "Santa Clara County and City of San Jose Housing Authority",
+  "help.housingHelp.vouchers.beniciaHA": "City of Benicia Housing Authority",
+  "help.housingHelp.vouchers.fairfieldHA": "City of Fairfield Housing Authority",
+  "help.housingHelp.vouchers.vacavilleHA": "City of Vacaville Housing Authority",
+  "help.housingHelp.vouchers.vallejoHA": "Housing Authority of the City of Vallejo",
+  "help.housingHelp.vouchers.sonomaHA": "Housing Authority of the County of Sonoma",
+  "help.housingHelp.vouchers.santaRosaHA": "Housing Authority of the City of Santa Rosa",
+  "help.housingHelp.relatedHelp.title": "211 for Housing-Related Help in the Bay Area",
+  "help.housingHelp.relatedHelp.header": "Si estás en el área de la Bahía y necesitas ayuda para resolver tus dudas relacionadas con la vivienda, llama al 211 (311 en San Francisco) para ponerte en contacto con un especialista en asistencia personalizada. También puedes utilizar la If you are in the Bay Area and need help with your housing concerns, dial 211 (311 in San Francisco) to connect to a specialist for personalized assistance. You can also use the ",
```

Comment:
this is in spanish and should be in english

Reply:
Oddly enough it's not locally for me...I had fixed this. I'm going to merge this & then change, since I'm having that issue with my branch.